### ISSUE TITLE
Specify destination file extensions

### ISSUE THREAD

**Issue by svenyonson** [imagemin/imagemin#280]
Maybe I've missed the correct way to convert PNG to JPG, but the example I found noted that the resulting JPG files will have a PNG extension.  I made a minor local edit to `imagemin/index.js` that fixes my issue - if this is useful please incorporate it - and if I am doing it wrong, please let me know.

I changed one line in `imagemin/index.js`, line 28:

`path: opts.destExt ? replaceExt(dest, opts.destExt) : (fileType(buf) && fileType(buf).ext === 'webp') ? replaceExt(dest, '.webp') : dest`

and I use it as follows: 

```
function processBook(book) {
	imagemin(['images/'+book+'/b/*.png'], 'images/'+book+'/t', {
	    plugins: [
	        pngToJpeg({quality: 80})
	    ],
	    destExt: '.jpg'
	}).then((files) => {
	    console.log('PNGs converted to JPEGs:');
	    for (f in files) {
	    	console.log(" - "+files[f].path);
	    }
	});
} 
```
Basically, I just added this options parameter: `destExt: '.jpg'`
